IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -1959,77 +1963,77 @@
 					"name": "Inventory and Order Management",
```

Comment:
Please name examples in this part like they are named in Help. If an example includes multiple requests, name as follows: `<Example name in help> <step number>: <brief description>`. For example: Creation of a Pro Forma Invoice 1: Create a Project
Same for the Organization part.

Reply:
You have not updated example names in the Inventory and Order Management section.